File: GRLTest/IndicatorconnecttoelE/IndicatorconnecttoelE/IndicatorconnecttoelE.java
public class Model{
	public double IndicatorconnecttoelE(double Indicator3){
		double expr = ((Indicator3 <= -100.0) ? (
   25.0
)
: ((Indicator3 <= 200.0) ? (
   1.0*Math.min(100.00000000000000, 12.5*Math.abs(0.00333333333333333*Indicator3 - 0.666666666666667) + 12.5)
)
: ((Indicator3 < 500.0) ? (
   1.0*Math.max(0, -12.5*Math.abs(0.00333333333333333*Indicator3 - 0.666666666666667) + 12.5)
)
: (
   0
))));
		return expr;
	}
}

File: GRLTest/IndicatorconnecttoelE/IndicatorconnecttoelE/IndicatorconnecttoelE.py
def IndicatorconnecttoelE( Indicator3):
	expr = ((25.0) if (Indicator3 <= -100.0) else (((1.0*min(100.0, 12.5*abs(0.00333333333333333*Indicator3 - 0.666666666666667) + 12.5)) if (Indicator3 <= 200.0) else (((1.0*max(0, -12.5*abs(0.00333333333333333*Indicator3 - 0.666666666666667) + 12.5)) if (Indicator3 < 500.0) else (((0) if (True) else None)))))))
	return expr
